README.md

```diff
@@ -1,23 +1,39 @@
+# DocTester
 
-Markdown file tester
-# this is a sample file to test feature.
-## POC file
+This library is used to test the code samples in the docments. Using this library you can always ensure that code samples in docments always work and avoids issues created for document update.
 
-```js
-import { add } from './add';
-add(4, 5) // equals: 9;
-add(4, 5) // not-equals: 6;
-add(2, 3) // throws;
+## Installation
+```sh
+npm install sparshithNR/doc-tester#master --save-dev
+```
+## Usage
+### From commandline
+```sh
+node_modules/.bin/doc-tester
 ```
+#options
+1. -f (--file)
+> File name/path which is to be tested. Defaulted to `README.md` in the root.
+2. -d (--debug)
+> Setting this to true will not remove the test file generated by parsing the documentation file. Defaulted to `false`.
+3. --inspect (--inspect-brk)
+> This will enable you to put debugging break points in test file. Defaulted to `false`.
```

Comment:
- [ ] add link to [node inspector](https://nodejs.org/en/docs/guides/debugging-getting-started/#enable-inspector
```suggestion
> Runs tests w/ the node `--inspect` option, allowing a debugger to be attached. (default: `false`)
```